tests/test_cli.py

```diff
@@ -0,0 +1,140 @@
+"""
+Collection of test cases to test the dj cli
+"""
+
+import json
+import subprocess
+import pytest
+import datajoint as dj
+from . import CONN_INFO_ROOT, PREFIX
```

Comment:
pytests not working in Dev Container with new pytest framework. Will need to migrate dependencies like `CONN_INFO_ROOT` to the respective `pytest.fixture`s.

```console
vscode ➜ /workspaces/datajoint-python (hidden-1091-continued) $ pytest tests/test_cli.py
========================================================= test session starts ==========================================================
platform linux -- Python 3.11.4, pytest-8.2.2, pluggy-1.5.0
rootdir: /workspaces/datajoint-python
plugins: Faker-28.4.1, cov-5.0.0
collected 0 items / 1 error                                                                                                            

================================================================ ERRORS ================================================================
__________________________________________________ ERROR collecting tests/test_cli.py __________________________________________________
ImportError while importing test module '/workspaces/datajoint-python/tests/test_cli.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/local/lib/python3.11/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/test_cli.py:9: in <module>
    from . import CONN_INFO_ROOT, PREFIX
E   ImportError: cannot import name 'CONN_INFO_ROOT' from 'tests' (/workspaces/datajoint-python/tests/__init__.py)
======================================================= short test summary info ========================================================
ERROR tests/test_cli.py
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
=========================================================== 1 error in 0.20s ===========================================================
```